functors task (1)/student/include/impl.h

```diff
@@ -8,6 +8,16 @@
  */
 struct BeerOrganizer
 {
+    int x = 1;
+
+    BeerBrand operator()()
+    {
+       int currentBeer = x;
+       if(x < 6) x++;
+       else x = 1;
```

Comment:
don't write two expressions in one line
also, follow this rule:
https://wiki.c2.com/?AlwaysUseBracesOnIfThen

Reply:
fixed